#!/usr/bin/env python3

PACKAGE = "shifted_line_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# param, data_type, 0, description, defaut, min, max

# sunny
gen.add("light_low", int_t, 0, "light low", 200, 0, 255)
gen.add("sat_low", int_t, 0, "sat low", 255, 30, 255)

# overcast
# gen.add("light_low", int_t, 0, "light low", 0, 0, 255)
# gen.add("sat_low", int_t, 0, "sat low", 40, 30, 255)

# the third parameter must match this file's name
exit(gen.generate(PACKAGE, "yellow_line", "YellowLine"))
